src/open_inwoner/openzaak/clients.py

```diff
@@ -38,7 +40,18 @@
 logger = logging.getLogger(__name__)
 
 
-class ZakenClient(APIClient):
+class ZgwAPIClient(APIClient):
+    """A client for interacting with ZGW services."""
+
+    configured_from: Service
+    """The Service instance used to configure this client."""
```

Comment:
A second docstring? Seems unusual. Also, the attribute seems to me sufficiently clear, no comment necessary.